build.sbt

```diff
@@ -1,10 +1,7 @@
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
 val V = new {
-  val scala_3_0 = "3.0.0-RC1"
-  val scala_2_13 = "2.13.5"
-  val scala_2_12 = "2.12.13"
-  val scalaVersion :: crossScalaVersions = List(scala_3_0, scala_2_13, scala_2_12)
+  val scalaVersion :: crossScalaVersions = List("2.13.5", "3.0.0-RC1")
```

Comment:
I think this will fail the build, since now we're trying to cross-compile for dotty?

Reply:
Hm, it doesn't fail it locally, but very likely there's still something I don't understand here

Reply:
Are you cross-compiling (i.e. using `+compile`) locally?

Reply:
I think the GH workflow is green because we're explicitly setting the scala version with `++$SCALA_VERSION ...`, and there's only 2.13.5 left in the matrix at this point, so it happens to succeed.

Reply:
Okay, so should I just drop Scala 3 too then?

Reply:
We're not cross-compiling for scala 3 currently, and we're no longer using it for the build itself - should we just remove references to it altogether for now?